frontend/src/modules/history/actions.js

```diff
@@ -102,25 +103,57 @@ export function updateStatus(
     ignoreWarnings: ?boolean,
 ): Function {
     return async dispatch => {
-        const results = await updateStatusOnServer(change, translation, resource, ignoreWarnings);
+        const results = await updateStatusOnServer(
+            change, translation, resource, ignoreWarnings
+        );
+
+        // Show a notification to explain what happened.
+        if (results.translation) {
+            let statusChange = change;
+            if (change.endsWith('e')) {
+                statusChange += 'd';
+            }
+            else {
+                statusChange += 'ed';
+            }
```

Comment:
This is hard to read. Especially if you consider we should make notifications localizable.